index.html

```diff
@@ -271,13 +273,48 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pt'>
+
+				<p>Nós somos uma comunidade de indivíduos que têm interesse significativo no desenvolvimento e na saúde da World Wide Web (“a Web”). <strong>Nós estamos profundamente preocupados</strong> sobre o Accelerated Mobile Pages (“AMP”), um projeto do Google que supostamente procura melhorar a experiência do usuário na Web.</p>
+
+				<p>Na verdade, a AMP mantém os usuários no domínio do Google e desvia o tráfego de outros sites para benefício do Google. Em uma escala de bilhões de usuários, isso tem o efeito de <strong>reforçar ainda mais o domínio do Google na Web.</strong></p>
+
+				<p>Reconhecemos o problema das páginas da Web serem lentas para carregar, em relação a tecnologias alternativas, como <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> e <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Os editores (especialmente nos meios de comunicação) já enfrentaram escolhas difíceis e pouco incentivos, levando a decisões ruins e por consequência uma terrível experiência para os usuários.</p>
+
+				<p>Os motores de busca estão numa posição poderosa para exercer influência que resolva esse problema. No entanto, o Google escolheu criar uma posição privilegiada na parte superior de seus resultados de busca (para artigos) e um ícone de "relâmpago" (para todos os tipos de conteúdo), que só são acessíveis para editores que usam a tecnologia controlada pelo Google, atendidos pelo mesmo a partir de suas infraestruturas, em um URL do Google e colocado dentro de uma experiência de usuário controlada pelo próprio Google. (<a href='https://developers.google.com/search/docs/guides/about-amp'>fonte</a>)</p>
+
+				<p>O formato AMP em si não é um problema, mas dois aspectos de sua implementação reforçam a posição do Google como uma plataforma padrão para conteúdos, já que o Google busca impulsionar a incorporação de AMP com criadores de conteúdo:</p>
```

Comment:
In “incorporação de AMP com criadores de conteúdo” replace  “com” with “por” or “entre”.